modular_skyrat/modules/goofsec/code/sol_fed.dm

```diff
@@ -102,17 +102,18 @@ GLOBAL_LIST_INIT(call911_do_and_do_not, list(
 				[GLOB.call_911_msg]"
 			announcer = "Sol Federation Marshal Department"
 			poll_question = "The station has called for the Marshals. Will you respond?"
-		if(EMERGENCY_RESPONSE_FIRE)
+		if(EMERGENCY_RESPONSE_ATMOS)
 			team_size = 8
-			cops_to_send = /datum/antagonist/ert/request_911/fire
-			announcement_message = "Crewmembers of [station_name()]. this is the Sol Federation. We've recieved a request for immediate firefighting support, and we are \
-				sending our best firefighters to support your station.\n\n\
-				If the first responders request that they need SWAT support to do their job, or to report a faulty 911 call, we will send them in at additional cost to your station to the \
+			cops_to_send = /datum/antagonist/ert/request_911/atmos
+			announcement_message = "Crewmembers of [station_name()]. this is the Sol Federation's 811 dispatch. We've recieved a report of stationwide structural damage, atmospherics loss, fire, or otherwise, and we are \
+				sending a Breach Control team to support your station.\n\n\
+				If the Breach Control team requests that they need SWAT protection to do their job, or to report a faulty 811 call, we will send them in at additional cost to your station to the \
 				tune of $20,000.\n\n\
 				The transcript of the call is as follows:\n\
 				[GLOB.call_911_msg]"
-			announcer = "Sol Federation Fire Department"
-			poll_question = "The station has called for the fire department. Will you respond?"
+			announcer = "Sol Federation 811 Dispatch - Breach Control"
+			poll_question = "The station has called for a Breach Control team. Will you respond?"
+			cell_phone_number = "811"	//Flavor - Call before you dig (Or in this case, blow a hole in your station)
```

Comment:
Leave this as 911, because this is the ID assigned for the cell phones and this would make them unable to communicate with the SWAT guys if they get called